openapi.yaml

```diff
@@ -356,3 +503,202 @@ components:
           type: string
           enum: [positions]
           example: 'positions'
+
+    ShiftListResponse:
+      type: object
+      properties:
+        data:
+          type: array
+          items:
+            $ref: '#/components/schemas/ShiftResponse'
+        included:
+          type: array
+          items:
+            $ref: '#/components/schemas/LocationResponse'
+
+    ShiftResponse:
+      type: object
+      properties:
+        id:
+          type: string
+          example: '1'
+        type:
+          type: string
+          example: 'shifts'
+        attributes:
+          $ref: '#/components/schemas/ShiftAttributes'
+        relationships:
+          type: array
+          items:
+            $ref: '#/components/schemas/LocationsRelationship'
+      required:
+        - id
+        - type
+
+    ShiftAttributes:
+      type: object
+      properties:
+        status:
+          type: string
+          example: 'open'
+        starts_at:
+          type: string
```

Comment:
[dust] Does the OpenAPI spec do anything to specify we want an ISO timestamp separate from a generic string type?

I thought I remembered something like that but I might be thinking of JSON schema instead so figured I'd ask in case there was something obvious in the docs we could use to better highlight the format we want.

Reply:
I think this should be:
```
        starts_at:
          type: string
          format: date-time
          example: '2020-01-01T06:00:00.000000Z'
```